web/static/js/ui.js

```diff
@@ -0,0 +1,8 @@
+$('.message .close')
```

Comment:
I rejiggered the alert boxes to be closeable and use Semantic styles[here](https://github.com/thinkthroughmath/dev_wizard/pull/42/files/7d624abc211841db0284cedb9b9897073a0b1056#diff-4e4c242ad765559995e9844dd68aa05aR44).